docs/getting-started.md

```diff
@@ -78,9 +78,13 @@ You can now run a mining node:
 radicle-registry-node --miner <address>
 ```
 
-`TODO: You can see mining in the logs`
+If your node successfully mined a block you will see a log line like this
+```
+15:50:13.041 INFO radicle_registry_node::service  Imported own block #1322 (0xefd1…4445)
```

Comment:
Not important but is there really a double space there?

```suggestion
15:50:13.041 INFO radicle_registry_node::service Imported own block #1322 (0xefd1…4445)
```

Reply:
Yes. It aids readability a bit.